Index: test/GitDemo01ClassTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test/GitDemo01ClassTest.java	(date 1549528797000)
+++ test/GitDemo01ClassTest.java	(date 1549530019376)
@@ -43,4 +43,10 @@
         int result = myObject.add(1890, 4);
         assertTrue(result == 1894);
     }
+    @Test
+    public void testZeljkaSantrac() {
+        GitDemo01Class myObject =new GitDemo01Class ();
+        int result= myObject.alwayszero(30, 31);
+        assertTrue(result==1988);
+    }
 }
\ No newline at end of file
